features/theme-install.feature

```diff
@@ -125,3 +125,35 @@ Feature: Install WordPress themes
       """
       twentyeleven
       """
+
+  Scenario: Can't install theme that requires a newer version of WordPress
+    Given a WP install
+
+    When I run `wp core download --version=6.4 --force`
+    And I run `rm -r wp-content/themes/*`
+
+    And I try `wp theme install twentytwentyfive`
+    Then STDERR should contain:
+    """
+    Warning: twentytwentyfive: This theme does not work with your version of WordPress.
+    """
+
+    And STDERR should contain:
+    """
+    Error: No themes installed.
+    """
+
+  @less-than-php-7.4 @require-wp-5.6
```

Comment:
I think these PHP version related tests are always going to be flaky. It is hard to find something that has a higher PHP requirement and doesn't also have a high WP requirement (meaning the test can't run on older versions). Also I suspect these requirements are going to change over time, where an upstream change can break it.

Outside of actually uploading our own test plugin to wp.org where we can artificially  set requirements for this test, I wonder if it just better not to have a test for this in favor of future stability.

Reply:
It would be easier if the API had requirements info even for older versions, but alas.

Let's keep it for now and see how flaky it will actually become.